src/platforms/linux/daemon/wireguardutilslinux.cpp

```diff
@@ -675,7 +675,8 @@ void WireguardUtilsLinux::nlsockHandleNewlink(struct nlmsghdr* nlmsg) {
   // Check for the interface going up.
   if ((diff & IFF_UP) && (msg->ifi_flags & IFF_UP)) {
     logger.info() << "Wireguard interface is UP";
-    setupWireguardRoutingTable();
+    setupWireguardRoutingTable(AF_INET);
+    setupWireguardRoutingTable(AF_INET6);
```

Comment:
The `inet` familly already includes both IPv4 and IPv6. Why is this necessary?

Reply:
In this case it does not, `AF_INET` only covers IPv4 when it comes to the routing table.

Reply:
TIL!